openstack/identity/v3/extensions/tokens/endpoint_location.go

```diff
@@ -0,0 +1,47 @@
+package tokens
```

Comment:
function V3EndpointURL(catalog *tokens3.ServiceCatalog, opts gophercloud.EndpointOpts) could not be used from `openstack/endpoint_location.go` so another method is required for that with different method signature, but implementing it in `openstack/endpoint_location.go` was not suitable so i moved it here.